crawler.py

```diff
@@ -756,6 +766,23 @@ def save(self, data, name='results.json'):
                 data, f, indent=2, sort_keys=True, separators=(',', ': '))
         self.logger.info("Saved data to %s", name)
 
+    def handle_clicking(self, domain):
+        # fetch the clickable elements on given domain
+        clickables = self.driver.find_elements_by_tag_name('a') + self.driver.find_elements_by_tag_name('button')
+
+        # strip the domain down so we can catch first party subdomains too
+        pattern = re.compile(r"https?://(www\.)?")
+        stripped_domain = pattern.sub('', domain).strip().strip('/')
```

Comment:
You don't need this if you simply pass in the domain (not the URL) in the first place.